test/gitWorkflow.spec.js

```diff
@@ -164,19 +162,23 @@ describe('gitWorkflow', () => {
         }
       `)
     })
+
     it('should checkout renamed file when hiding changes', async () => {
       const gitWorkflow = new GitWorkflow({
         gitDir: cwd,
         gitConfigDir: path.resolve(cwd, './.git'),
       })
+
       const origContent = await readFile('README.md')
       await execGit(['mv', 'README.md', 'TEST.md'])
       await appendFile('TEST.md', 'added content')
 
       gitWorkflow.partiallyStagedFiles = await gitWorkflow.getPartiallyStagedFiles()
       const ctx = getInitialState()
       await gitWorkflow.hideUnstagedChanges(ctx)
-      expect(await readFile('TEST.md')).toStrictEqual(origContent)
+
+      /** @todo `git mv` in GitHub Windows runners seem to add `\r\n` newlines in this case. */
+      expect(normalizeWindowsNewlines(await readFile('TEST.md'))).toStrictEqual(origContent)
```

Comment:
This might be a bug in Git, since newlines have been configured to `\n`, and still `git mv` seems to convert it to `\r\n`... Not sure, but this workaround seems to work.